pkg/apicast/apicast_option_provider.go

```diff
@@ -338,3 +381,19 @@ func (a *APIcastOptionsProvider) customEnvSecret(nn types.NamespacedName) (*v1.S
 
 	return secret, nil
 }
+
+func (a *APIcastOptionsProvider) validateTracingConfigSecret(nn types.NamespacedName) error {
+	secret := &v1.Secret{}
+	err := a.Client.Get(context.TODO(), nn, secret)
+
+	if err != nil {
+		// NotFoundError is also an error, it is required to exist
+		return err
+	}
+
+	if _, ok := secret.Data[TracingConfigSecretKey]; !ok {
+		return fmt.Errorf("Required secret key, %s not found", "config")
```

Comment:
minor: use `TracingConfigSecretKey` instead of raw `config`

Reply:
updated :+1: